### PR TITLE
Hack multilingual devas796

### PR DESCRIPTION
**Pull request by paolina92**

### CODE REVIEW

File: assembl/static2/js/app/components/debate/common/header.jsx [hunk 1]
```diff
@@ -12,12 +12,19 @@ class Header extends React.Component {
     const { locale } = this.props.i18n;
     const isPhaseCompleted = getIfPhaseCompletedByIdentifier(debateData.timeline, identifier);
     const closedPhaseName = getPhaseName(debateData.timeline, identifier, locale).toLowerCase();
+    const titlesArray = title.split('#!');
+    let localizedTitle = '';
+    titlesArray.forEach((t) => {
+      if (t.split('$!')[1] === locale) {
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/components/debate/common/header.jsx]: ```if (locale in t.split('$!')[1] { ... }```. At least the hack would be more resilient to whitespace or other chars in between.

File: assembl/static2/js/app/pages/debateThread.jsx [hunk 2]
```diff
@@ -10,9 +10,25 @@ import Timeline from '../components/debate/navigation/timeline';
...
+  const thematicsWithLocales = 'ideas' in data ? data.ideas : null;
+  // This hack should be removed when the TDI's admin section will be done.
+  // We need it to have several langString in the idea's title
+  const thematics = [];
+  let titlesArray = [];
+  thematicsWithLocales.forEach((thematic) => {
+    if (thematic.title) {
+      titlesArray = thematic.title.split('#!');
+      titlesArray.forEach((title) => {
+        const titleLocale = title.split('$!')[1];
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/pages/debateThread.jsx]: ```title.split('$!')[1].trim()```
Remove any whitespace that the admins may inadvertently put in the strings.

File: assembl/static2/js/app/pages/debateThread.jsx [hunk 3]
```diff
@@ -10,9 +10,25 @@ import Timeline from '../components/debate/navigation/timeline';
...
+        if (titleLocale === lang) {
+          thematics.push({ ...thematic, title: title.split('$!')[0] });
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/pages/debateThread.jsx]: Isn't the `0`th index the locale? The `1`st index, the content?

Comment (paolina92) [reply assembl/static2/js/app/pages/debateThread.jsx]: Nope.

File: assembl/static2/js/app/pages/debateThread.jsx (hunk 2 above)
Comment (paolina92) [reply assembl/static2/js/app/pages/debateThread.jsx]: We don't want to remove spaces in the title

**Pull request merged**